jme3-core/src/main/java/com/jme3/scene/BatchNode.java

```diff
@@ -387,7 +381,6 @@ private void mergeGeometries(Mesh outMesh, List<Geometry> geometries) {
         int maxWeights = -1;
 
         Mesh.Mode mode = null;
-        float lineWidth = 1f;
```

Comment:
Why was this line removed?

Reply:
The variable isn't referenced anywhere else in the function, it's simply dead code.